website/lib/changelog/generate-component-changelog-entries.mjs

```diff
@@ -99,6 +100,75 @@ const updateComponentVersionHistory = (componentChangelogEntries, version) => {
   });
 };
 
+const updateComponentFrontMatter = (componentChangelogEntries, version) => {
+  Object.keys(componentChangelogEntries).forEach((componentName) => {
+    const indexPath = `${allComponentsPath[componentName]}/index.md`;
+
+    if (fs.existsSync(indexPath)) {
+      // Fetch the index markdown file
+      const indexContent = fs.readFileSync(indexPath, 'utf8');
+
+      // Parse the index file content
+      const parsedFrontMatter = matter(indexContent);
+
+      // Update the front matter
+      if (!parsedFrontMatter.data.status) {
+        parsedFrontMatter.data.status = {};
+      }
+      if (
+        parsedFrontMatter.data.status.added !== version &&
```

Comment:
Question: going forward, will we still need to manually add the `added` tag?

Reply:
indeed. either add it manually, or change it from Updated to Added where needed. We can try to add some logic there to distinguish additions from updates, but it's not going to be a clean cut and wasn't sure it's worth the effort.

Reply:
Makes sense! We add stuff much less often.